app/models/note.rb

```diff
@@ -2,7 +2,16 @@ class Note < ApplicationRecord
   belongs_to :user
   validates_presence_of :title, :content, :date
 
-  def self.grouped_by_month(user)
-    Note.where(user_id: user.id).order(date: :DESC).group_by { |note| note.date.beginning_of_month }
+  def self.grouped_by_month
+    order(date: :DESC).group_by { |note| note.date.beginning_of_month }
```

Comment:
If the `Note` requires a date to be present, this is fine. As the code reviewer I look out for `methods.with.a.lot.of.dots` which are prone to `NilpointerExceptions`.

You could add some syntactic sugar:
```ruby
class Note
  delegate :beginning_of_month, to: :date
end
```